### DISCUSSION TITLE
Outpoint in resolution from closedchannels doesn't exist in blockchain

### DISCUSSION THREAD

**Discussion by kafaichoi** [lightningnetwork/lnd#7729] [Q&A] [answered]
Hello all. I noticed that there are *sometimes* resolutions from `lncli closedchannels` having outpoint that doesn't exist in the blockchain. They are usually force closed locally. An example of this resolution is as follow. Isn't this outpoint supposed to be the closing transaction vout? Or Did I misunderstand sth?

```
      {
        "resolution_type": "INCOMING_HTLC",
        "outcome": "TIMEOUT",
        "outpoint": {
          "txid_bytes": "120b5f446342cefcac0a6d237b6ea6d4395d46e320981ec4d1bbfb18332a2672",
          "txid_str": "72262a3318fbbbd1c41e9820e3465d39d4a66e7b236d0aacfcce4263445f0b12",
          "output_index": 0
        },
        "amount_sat": "34923",
        "sweep_txid": ""
      },
```

Many thanks

**Comment by ellemouton** [comment]
Hi @bruteforcecat - if they are local force closes then any HTLC output on your commitment transaction will have a second stage transaction. So the output above is referencing the second stage htlc which might not yet be confirmed

**Reply by kafaichoi** [reply]
Thank you for your reply.  So the outpoint in resolution(INCOMING_HTLC, TIMEOUT) refer to the sweep transaction by remote? But looking at the data, sometime it's simply pointing to closing tx as well. Why is that?

(btw this force close have made more than few months ago. )

**Reply by ellemouton** [reply]
> refer to the sweep transaction by remote?

no - I think they refer to the second level HTLC timeout transaction. 

> sometime it's simply pointing to closing tx as well

For other `INCOMING_HTLC` ones? I would expect the `COMMIT` ones to refer to the closing tx. 

> (btw this force close have made more than few months ago. )

hmmm ok that does seem strange. Can you find the actual force close on-chain? and then if so, can you check if the HTLC output with the `34923` output has been spent (by 2 transactions in a row)? 
If you feel comfortable you can also just DM me the TXID of the fore close tx and then I can investigate it a bit more. 
Im 'elle' on the LND developer Slack

**Comment by ellemouton** [comment, marked as answer]
Ok, I investigated a bit more and now have a better understanding: 

So what happened is that at the time of force closing, there was an incoming HTLC on the commitment transaction that your node did not yet have the pre-image for. And so eventually, the remote node was able to sweep the HTLC via the timeout path.
If you were to have had the pre-image, you would have broadcasted the second level tx with the TXID mentioned there. But since you did not have the pre-image, your node could not broadcast that second level tx which is why it is not found on-chain. 

Hope that makes sense!

**Comment by ghost** [comment]
I'm having a similar problem but it's not related to channel close... it's an anchor/sweep transaction from inside the node. My nodes mempool has no knowledge of it, and the outpoint doesn't seem to exist either. Is there a wallet that I can import the Aezeed to that explores taproot addresses? I want to keep my node running but re-sweep this transaction.

**Reply by ellemouton** [reply]
@godSaysHODL  - im not sure your issue report belongs here? This was just a question around interpreting the contents of an output in `listchannels`. Perhaps consider opening an issue or finding on that looks similar and commenting on there?
